src/client/components/search-button/style.scss

```diff
@@ -0,0 +1,18 @@
+.superlink {
+  width: 38px;
+  height: 38px;
+  margin-left: 10px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 28px;
+  border-radius: 2px;
+  &.google {
+    background-image: url("../../assets/google.svg");
+    background-color: var(--dashboard-bg-darkest);
```

Comment:
You can move `bg-color` to shared style.

Reply:
yep -- they were differnet at first :p done!